app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/actions.ts

```diff
@@ -219,7 +219,8 @@ export const getSubmissionsByFormat = async ({
         }
       );
       let sorted: Answer[];
-      if (allowGroupsFlag) {
+      const formGroups = fullFormTemplate.form.groups ?? [];
```

Comment:
Can use 

```
formHasGroups
```

Believe

Since conditional rules the groups object can exist but will be {} (object with no keys)